### PR TITLE
feat: added more chains in a51 finance v3 tvl

### PR DESCRIPTION
**Pull request by HamzaBhatti125**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
A51 Finance V3

### CODE REVIEW

**Comment by HamzaBhatti125** [comment]
I didn't do any changes in utils/handleError.js as the changes are showing due to the eslint formatting in order to debug the error details

**Comment by llamatester** [comment]
The adapter at projects/a51-finance-v3 exports TVL: 
        
 
 
    polygon-staking           73.63 k
    staking                   73.63 k
    arbitrum                  4.92 k
    blast                     26
    bsc                       13
    polygon                   12
    base                      9
    optimism                  8
    scroll                    7
    
    total                    5.00 k

File: utils/handleError.js
Comment (g1nt0ki) [inline utils/handleError.js:1]: please revert change to this file

Comment (HamzaBhatti125) [reply utils/handleError.js:1]: @g1nt0ki done!

**Comment by llamatester** [comment]
Error while running adapter at projects/a51-finance-v3: 
        
 
  
    
    
    Error: Promise pool failed! 
     PromisePoolError: Failed to call: function tryAggregate(bool requireSuccess, tuple(address target, bytes callData)[] calls) payable returns (tuple(bool success, bytes returnData)[] returnData) target: 0xca11bde05977b3631167028862be2a173976ca11 params: false, 0x6F2b186e9392042B1edE2D1D1706a3DC4a4725d8,0x3ff34eb9a08ba6f18a81b51b3feaf757d2831ecd1eb373040668ea9ece51cdc519378708,0x6F2b186e939204... on chain: [bsc] rpc: https://rpc.ankr.com/bsc, https://bsc-dataseed4.binance.org, https://bsc-dataseed.binance.org, https://bsc-dataseed1.defibit.io, https://bsc-dataseed1.ninicoin.io, https://bsc-dataseed2.defibit.io, https://bsc-dataseed2.ninicoin.io, https://binance.llamarpc.com, https://bsc-dataseed.bnbchain.org, https://bsc-dataseed3.defibit.io, https://bsc-dataseed4.defibit.io, https://bsc-dataseed3.ninicoin.io, https://bsc-dataseed4.ninicoin.io, https://bsc-dataseed1.bnbchain.org, https://bsc-dataseed2.bnbchain.org, https://bsc-dataseed3.bnbchain.org, https://bsc-dataseed4.bnbchain.org, https://rpc-bsc.48.club, https://koge-rpc-bsc.48.club, https://endpoints.omniatech.io/v1/bsc/mainnet/public, https://bsc-mainnet.nodereal.io/v1/64a9df0874fb4a93b9d0a3849de012d3, https://rpc.ankr.com/bsc, https://bscrpc.com, https://binance.nodereal.io, https://bsc-mainnet.rpcfast.com?api_key=xbhWBI1Wkguk8SNMu1bvvLurPGLXmgwYeC4S6g2H7WdwFigZSmPWVZRxrskEQwIf, https://1rpc.io/bnb, https://bsc.rpc.blxrbdn.com, https://bsc.blockpi.network/v1/rpc/public, https://bnb.api.onfinality.io/public, https://bsc-rpc.publicnode.com, https://bsc-mainnet.public.blastapi.io, https://bsc.meowrpc.com, https://bsc.drpc.org, https://public.stackup.sh/api/v1/node/bsc-mainnet, https://api.tatum.io/v3/blockchain/node/bsc-mainnet  call reverted, reason: fragment.format is not a function

**Comment by g1nt0ki** [comment]
@HamzaBhatti125 I think, it was RPC issue, fixed it here: https://github.com/DefiLlama/DefiLlama-Adapters

**Comment by g1nt0ki** [comment]
*ignore rpc issue, added new chains: https://github.com/DefiLlama/DefiLlama-Adapters/commit/ce69e539f573313635d3527de580b8cf9d5d0425

**Pull request closed without merging**